libs/telemetry/src/main/java/org/opensearch/telemetry/metrics/MetricsRegistry.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.telemetry.metrics;
+
+import java.io.Closeable;
+
+/**
+ * MetricsRegistry helps in creating the metric instruments.
+ */
+public interface MetricsRegistry extends Closeable {
```

Comment:
`MetricsRegistry` and `MetricsTelemetry` are same . Can we just use `MetricsTelemetry` ?

Reply:
If you see MetricsRegistry implementations are DefaultMetricsRegistry and WrappedMetricsRegistry which are very much specific to the OpenSearch but MetricsTelemetry implementations are specific to the particular tools we are using like OpenTelemetry here.

Reply:
We could keep `MetricsTelemetry` and `MetricsRegistry`  but since those are indeed identical, we could simplify the abstractions a bit: 

```
public interface MetricsRegistry extends MetricsRegistry {
}
```